### PR TITLE
feat(datalog-console): respond to datalog-console messages in browser

### PR DESCRIPTION
**Pull request by groundedSAGE**
Adds support for the [Datalog Console chrome extension](https://chrome.google.com/webstore/detail/datalog-console/cfgbajnnabfanfdkhpdhndegpmepnlmb)

### CODE REVIEW

**Review by tangjeff0 — APPROVED** [review]
Extension doesn't seem to recognize datalog DB, but it does work for browser. DB is recognized when using the Homebase Todo App.

# Athens

image


# Homebase Todo App

image

File: src/cljs/athens/core.cljs [hunk 1]
```diff
@@ -98,6 +117,7 @@
   (style/init)
   (stylefy/tag "body" style/app-styles)
   (listeners/init)
+  (init-datalog-console)
```
Comment (arkRedM) [inline src/cljs/athens/core.cljs:120]: I think we should put this under dev/debug flag.

**Comment by groundedSAGE** [comment]
@tangjeff0 that is strange it appears to show up when the page is refreshed but not on the initial load. Might have to periodically check for the attribute that is used for notification.

File: src/cljs/athens/core.cljs (hunk 1 above)
Comment (groundedSAGE) [reply src/cljs/athens/core.cljs:120]: Yea I'm 50/50 on whether it should be a debug only option. I kind of like the idea of giving users an admin console into their own database. Only downside I can see is some resource usage overhead and the need for more security as the console becomes more powerful.

Comment (tangjeff0) [reply src/cljs/athens/core.cljs:120]: I think this can be available to everyone.

**Pull request merged**